packages/graphql-hooks/package.json

```diff
@@ -35,6 +35,7 @@
     "react": "^17.0.0"
   },
   "dependencies": {
+    "@aws-crypto/sha256-browser": "^2.0.1",
```

Comment:
Is there a specific reason why you used the browser compatible version here? I see that there is a "regular" js version [sha256-js](https://github.com/aws/aws-sdk-js-crypto-helpers/tree/master/packages/sha256-js) as well as [sha256-universal](https://github.com/aws/aws-sdk-js-crypto-helpers/tree/master/packages/sha256-universal).

Reply:
The only reason is that the browser version has a fallback for when `window.crypto.subtle` is not present. I didn't spot the universal though - that would work too but it's a little bigger lib